### ISSUE TITLE
Implement accessibility labels for playback controls and result options

### ISSUE THREAD

**Issue by Piciok** [nukeop/nuclear#24]
Currently the **buttons allowing to play/download a track found as well as the playback controls lack the accessibility labels allowing screenreaders to describe them to blind users. Tested under Windows and NVDA 2017.1

**Comment by nukeop** [comment]
Hey, I added the labels you requested. Not sure if they're not too verbose, or too vague, so let me know if there's anything wrong. Also please let me know if there's anything more I can do to improve accessibility for you and other users.
